bin/install

```diff
@@ -1,10 +1,16 @@
 #!/usr/bin/env php
 <?php
 
+define('APPLICATION_ENV', (getenv('APPLICATION_ENV') ?: 'development'));
```

Comment:
I thought you will use the Environment class? That would already have probably all you need. Or was there something blocking you from using it?

Reply:
I am confused. Haven’t we agreed on using the approach with manually define ing the constants?
But yes, there is a problem actually.
Environment class itself required the `APPLICATION` constant to be defined.
And it’s mandatory.